Makefile

```diff
@@ -246,7 +246,7 @@ test-capi-llvm-native:
 	cargo test --manifest-path lib/c-api/Cargo.toml --release \
 		--no-default-features --features deprecated,wat,native,llvm,wasi $(capi_default_features) -- --nocapture
 
-test-capi-examples:
+test-capi-examples: package-capi
```

Comment:
Oops!

Reply:
@syrusakbary i did that on purpose: the examples expect to find the `include/` and `lib/` directory in `WASMER_DIR`.

After the library has been built by the regular tests, it's in `target/release` without the expected structure.

I know this is not ideal: it makes the `test-capi-examples` target implicitely dependant to `build-capi`

Reply:
Oh! The Oops was meant for myself for forgetting that :P